src/Microsoft.Extensions.OptionsModel/IOptionsChangeTracker.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.Extensions.OptionsModel
+{
+    public interface IOptionsChangeTracker<out TOptions>
```

Comment:
https://github.com/aspnet/specs/blob/master/notes/reloadable_options.md#proposed-api-changes

This also deviates from what we had spec'ed out. Why this approach over the other? I think it's probably simpler this way but curious.